static/app/gettingStartedDocs/python/django.tsx

```diff
@@ -39,31 +39,15 @@ sentry_sdk.init(
 `;
 
 const onboarding: OnboardingConfig = {
-  introduction: () =>
-    tct('The Django integration adds support for the [link:Django Web Framework].', {
-      link: <ExternalLink href="https://www.djangoproject.com/" />,
-    }),
   install: (params: Params) => [
     {
       type: StepType.INSTALL,
-      description: tct(
-        'Install [code:sentry-sdk] from PyPI with the [sentryDjangoCode:django] extra:',
-        {
-          code: <code />,
-          sentryDjangoCode: <code />,
-        }
-      ),
+      description: tct('Install [code:sentry-sdk] from PyPI:', {
+        code: <code />,
+      }),
       configurations: [
         {
-          description: params.isProfilingSelected
-            ? tct(
-                'You need a minimum version [codeVersion:1.18.0] of the [codePackage:sentry-python] SDK for the profiling feature.',
-                {
-                  codeVersion: <code />,
-                  codePackage: <code />,
-                }
-              )
-            : undefined,
+          description: params.isProfilingSelected,
```

Comment:
I tested this locally and it seems fine, but maybe we want to remove this since we do not have a description anymore?
```suggestion
```